gadget/gadget_test.go

```diff
@@ -807,13 +807,21 @@ var (
 	}
 )
 
+func setEnclosingVolumeInStructs(vv map[string]*gadget.Volume) {
```

Comment:
this might be something to share via gadgettest, no?

Reply:
Changed now